src/api/entities/Asset/NonFungible/NftCollection.ts

```diff
@@ -35,10 +51,6 @@ export class NftCollection extends BaseAsset {
       { getProcedureAndArgs: args => [transferAssetOwnership, { ticker, ...args }] },
       context
     );
-    this.controllerTransfer = createProcedureMethod(
-      { getProcedureAndArgs: args => [controllerTransfer, { ticker, ...args }] },
-      context
-    );
```

Comment:
Why `transferOwnership` is not defined on `BaseAsset`?

Reply:
I think I got mixed up with `controllerTransfer` which was mentioned to only work for FungibleAssets atm.

Agreed `transferOwnership` should be on `BaseAsset`